parachain/pallets/ethereum-beacon-light-client/Cargo.toml

```diff
@@ -0,0 +1,64 @@
+[package]
+name = "snowbridge-ethereum-beacon-light-client"
+description = "Snowbridge Beacon Light Client Pallet"
+version = "0.0.1"
+edition = "2018"
+authors = ["Snowfork <contact@snowfork.com>"]
+repository = "https://github.com/Snowfork/snowbridge"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+serde = { version = "1.0.130", optional = true }
+codec = { version = "2.2.0", package = "parity-scale-codec", default-features = false, features = ["derive"] }
+scale-info = { version = "1.0", default-features = false, features = ["derive"] }
+hex = { package = "rustc-hex", version = "2.1.0", default-features = false }
+hex-literal = { version = "0.3.1", optional = true }
+rlp = { version = "0.5", default-features = false }
+eth2_ssz = { version = "0.4.0" }
+eth2_ssz_derive = { version = "0.3.0" }
+eth2_ssz_types = { version = "0.2.1" }
+tree_hash = { version = "0.4.0" }
+tree_hash_derive = { version = "0.4.0" }
+milagro_bls = { git = "https://github.com/sigp/milagro_bls", tag = "v1.4.2" }
+
+frame-benchmarking = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.12", default-features = false, optional = true }
+frame-support = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.12", default-features = false }
+frame-system = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.12", default-features = false }
+sp-core = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.12", default-features = false }
+sp-std = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.12", default-features = false }
+sp-io = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.12", default-features = false }
+sp-runtime = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.12", default-features = false }
+
+snowbridge-core = { path = "../../primitives/core", default-features = false }
+snowbridge-ethereum = { path = "../../primitives/ethereum", default-features = false }
+
+[dev-dependencies]
+sp-keyring = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.12" }
+hex-literal = { version = "0.3.1" }
+snowbridge-testutils = { path = "../../primitives/testutils" }
+serde_json = "1.0.68"
+
+[features]
+default = ["std"]
+std = [
+    "serde",
+    "codec/std",
+    "scale-info/std",
+    "frame-support/std",
+    "frame-system/std",
+    "rlp/std",
+    "sp-core/std",
+    "sp-io/std",
+    "sp-runtime/std",
+    "sp-std/std",
+    "snowbridge-core/std",
+    "snowbridge-ethereum/std",
+]
```

Comment:
Need to add the std feature of milagro_bls here. So that when we want to compile the code to native machine code (not WASM) we can enable std feature and allow linking to the core std library.